catch-uncommitted.sh

```diff
@@ -39,7 +39,10 @@ if [ $catch_no_git -eq 1 ] && !(hash git 2> /dev/null); then
 elif ! git diff HEAD --exit-code ${ignore_eol} -- . "${extra_args[@]}" ; then
 	echo '** Uncommitted changes found (^^^ diff above ^^^) - FAIL **'
 	exit 1
-elif test -n "$(git ls-files --exclude-standard --others | tee /dev/tty)" ; then
+
```

Comment:
Missing `fi`?

Reply:
Yep, I should really stop trying to multitask while editing these files. :sweat: 
I might look into adding some simple tests that check stderr/stdout in a new PR.